lib/child-writer/index.js

```diff
@@ -192,7 +194,7 @@ exports.write = (image, drive, options) => {
 
     child.on('error', emitError)
 
-    child.on('close', (code) => {
+    child.on('exit', (code, signal) => {
```

Comment:
Does `signal` need to be handled here at all?

Reply:
I'm not sure I follow? Why wouldn't it be? The writer-proxy is also prone to being annihilated by a SIGKILL or so, or what am I missing?

Reply:
Hah, @jviotti beat me to it ;)

Reply:
Oh, that's about that one! I only saw your comment after I already pushed it, hence the great confusion :)